app/javascript/articles/Article.jsx

```diff
@@ -0,0 +1,406 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import {
+  articlePropTypes,
+  articleSnippetResultPropTypes,
+} from '../src/components/common-prop-types/article-prop-types';
+import {
+  organizationPropType,
+  tagPropTypes,
+} from '../src/components/common-prop-types';
+import { PodcastArticle } from './PodcastArticle';
+
+/* global timeAgo */
+
+// TODO: faking this from backend function asset_path. Need to get this in the frontend.
+const assetPath = relativeUrl => `/images/${relativeUrl}`;
```

Comment:
@ben @rhymes @mstruve, I'm not too familiar with Fastly.

As the comment states, faking this for now. Is there an equivalent on the client-side for `asset_path`, e.g. `<%= asset_path("comments-bubble.png") %>`. I imagine no as this generates a fastly URL. Just wondering how to best handle this as I need that asset in the Preact component among others. I could output in an ERB file a snippet of JS with the urls as an asset object potentially, e.g.

``` javascript
const assets = {
 "comments_bubble": "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/comments-bubble-9958d41b969a1620c614347d5ad3f270ab49582c1d9f82b617a6b4156d05dda0.png"
...
}
```

but that is a last resort. Any suggestions?

Reply:
I feel like there must be a Webpacker config to send asset pipeline paths to app/javascript eh?

Reply:
Good suggestion. I’ll see if there’s anything over there in webpacker land in regards to the asset pipeline.